.github/Dev/push-to-docker-bits.yaml

```diff
@@ -0,0 +1,33 @@
+name: Push to Docker bits
```

Comment:
Wanted to iterated faster - this was just for testing and is in a dev folder now